distkeras/evaluators.py

```diff
@@ -46,3 +49,33 @@ def evaluate(self, dataframe):
         validated_instances = cleaned.count()
 
         return float(validated_instances) / float(num_instances)
+
```

Comment:
Could you add an additional white line? This makes the boundary between classes more visible.